index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Itim&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Itim&family=Lobster&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+    
+</head>
+<body>
+    <header>
+        <h1>PANCAKES or CREPES or MLYNTSI</h1>
+    </header>
+    <main>
+        <img id="top_image" src="/images/Bez-nachinki.jpg" alt="pancakes on a plate"/>
+        <div id="ingredients">
+            <h3>You will need</h3>
+            <ul>
+                <li>4 eggs</li>
+                <li>1 glass of milk</li>
+                <li>1/2 teaspoon salt</li>
+                <li>1 (or 2) tablespoon sugar</li>
+                <li>10 ml vegetable oil</li>
+                <li>50 ml sparkling water (optional)</li>
+            </ul>
+        </div>
+        <div id="steps">
+            <h3>What to do with all that</h3>
+            <ol>
+                <li>Combine the ingredients Whisk to combine. </li>
+                <li>With an electric mixer, beat until well combined.</li>
+                <li>Heat a nonstick skillet over medium-high and spray with nonstick cooking spray.</li>
+                <li>Ladle the batter into the pan just to cover the bottom. Cook until the surface begins to bubble, then flip over to cook the other side.</li>
+                <li>Serve hot with desired toppings.</li>
+            </ol>
+        </div>
+        <img id="bottom_image" src="/images/carnival-3977209_1920.jpg" alt="pancakes on the table and a cat is looking at them"/>
+    </main>
+    <footer>
+        ENJOY YOUR MEAL :)!!!!!
```

Comment:
we can use a html special character for the smiley face https://www.science.co.il/internet/html/Smileys.php